### PR TITLE
[CPDLP-1723] Enable admin users to manage AB users on the service

### PR DESCRIPTION
**Pull request by mooktakim**
### Context

- Ticket: https://dfedigital.atlassian.net/browse/CPDLP-1723

### Changes proposed in this pull request

* New admin section "Appropriate bodies" to manage AB users
* Login with "admin@example.com"
* AB admin console hidden behind feature flag `appropriate_bodies`

### CODE REVIEW

File: app/views/admin/appropriate_bodies/users/delete.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<% content_for :title, "Delete user" %>
+<h1>Do you want to delete this user?</h1>
+<p class="govuk-!-font-weight-bold">Supplier user: <span class="govuk-!-font-weight-regular"><%= @appropriate_body_profile.user.full_name %></span></p>
+<%= form_for :confirmation, url: admin_appropriate_bodies_user_path(@appropriate_body_profile), method: :delete do |f| %>
+  <div class="govuk-button-group">
+    <%= f.govuk_submit "Delete", classes: "govuk-button--warning", "data-test": "delete-button" %>
```
Comment (peteryates) [inline app/views/admin/appropriate_bodies/users/delete.html.erb:6]:
```suggestion
    <%= f.govuk_submit "Delete", warning: true, "data-test": "delete-button" %>
```

**Comment by peteryates** [comment]
Code wise it looks ok to me. I notice that in the review app there are two user records listed that contain the same data (and they both change when one is edited). Should there be a distinct in there somewhere?

**Comment by mooktakim** [comment]
> Code wise it looks ok to me. I notice that in the review app there are two user records listed that contain the same data (and they both change when one is edited). Should there be a distinct in there somewhere?

Its the same user with multiple AB roles.  This list is not users but AB profiles for users.
There's larger work that we need to do to combine all users, AB's and DP's in one place we add/edit/delete user and then add/remove roles for that user.

**Comment by peteryates** [comment]
Could we add an ID or something to differentiate them? I think the rows being identical is likely to confuse people.

File: app/controllers/admin/appropriate_bodies/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module Admin
+  module AppropriateBodies
+    class UsersController < Admin::BaseController
+      skip_after_action :verify_authorized
```
Comment (cwrw) [inline app/controllers/admin/appropriate_bodies/users_controller.rb:6]: should `verify_authorized` and `verify_policy_scoped` be scoped to only/except index 🤔 ?

Comment (mooktakim) [reply app/controllers/admin/appropriate_bodies/users_controller.rb]: @cwrw 
Corrected the pundit policies 👍

**Review by cwrw — APPROVED** [review]

**Review by peteryates — APPROVED** [review]
Looks good!

**Pull request merged**